BE/Controllers/ClientsController.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Linq;
+using BE.BL.Cars.Create;
+using BE.BL.Cars.Edit;
+using BE.BL.Revisions.Create;
+using BE.BL.Revisions.Edit;
+using BE.Queries.Cars;
+using BE.Queries.Revisions.GetAllRevisionsByClientId;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace BE.Controllers
+{
+    [Route("api/[controller]")]
+    public class ClientsController
+    {
+
+        #region Ctor
+        private readonly MyContext _context;
+        public ClientsController(MyContext context)
+        {
+            _context = context;
+        }
+        #endregion
+
+        #region Get
+        [HttpGet("{clientId}/{carId}")]
+        public GetCarByIdResult GetCarById(long clientId,long carId)
+        {
+            
+            return _context.Set<GetCarByIdResult>().FromSqlRaw("EXEC [dbo].[usp_GetCarById] {0},{1}", carId,clientId).ToList().FirstOrDefault();            
+        } 
+
+        [HttpGet("{clientId}/cars")]
+        public List<GetAllCarsByClientIdResult> GetAllCarsByClientId(long clientId)
+        {
+            return _context.Set<GetAllCarsByClientIdResult>().FromSqlRaw("EXEC [dbo].[usp_GetAllCarsByClientId] {0}",clientId).ToList();            
+        }
+
+        [HttpGet("{clientId}/revisions")]
+        public List<GetAllRevisionsByClientIdResult> GetAllRevisionsByClientId(long clientId)
+        {
+
+            return _context.Set<GetAllRevisionsByClientIdResult>().FromSqlRaw("EXEC [dbo].[usp_GetAllRevisionsByClientId] {0}", clientId).ToList();
+        }
+
+        #endregion
+
+        #region Post
+        [HttpPost("{clientId}/Cars")]
```

Comment:
REST paths should always be **camelCase**

```suggestion
        [HttpPost("{clientId}/cars")]
```